pkgs/applications/misc/colorstorm/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, fetchFromGitHub, zig_0_9 }:
+zig_0_9.buildZigPackage rec {
```

```suggestion
{ lib, fetchFromGitHub, zig_0_9 }:

zig_0_9.buildZigPackage rec {
```

Also, try to adopt the rec-less approach of https://github.com/NixOS/nixpkgs/pull/119942